pkgs/development/python-modules/prox-tv/dependencies.patch

```diff
@@ -0,0 +1,11 @@
+index f100b35..448bbaf 100644
+--- a/prox_tv/prox_tv_build.py
++++ b/prox_tv/prox_tv_build.py
+@@ -109,6 +109,6 @@ ffi.set_source(
+     define_macros=[('NOMATLAB', 1)],
+     extra_compile_args=extra_compile_args,
+     extra_link_args=extra_link_args,
+-    libraries=['blas', 'lapack'],
++    libraries=['openblas', 'blas', 'lapack', 'gfortran'],
```

Comment:
should this be upsteamed?

Reply:
I have _no idea_, I'll open an issue just in case

Reply:
i don't think it should be upstreamed, as the patch I submitted makes their build fail, but I have no idea why.

Reply:
@jonringer I think this patch is needed because of [this pkg-config alias](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/libraries/science/math/openblas/default.nix#L144), but I'm not quite sure. @timokau I think some indirect dependencies are missing there?

Reply:
I think this code expects a thunk library called `blas` to be there, which would pull in the `gfortran` and `openblas` dependencies

Reply:
my bad, I'm all wrong and the fix is pretty darn simple

Reply:
I defaulted to blas / liblapack on darwin and openblas otherwise (see https://github.com/NixOS/nixpkgs/pull/45013)